src/main/scala/forms/DateWithDaysRule.scala

```diff
@@ -0,0 +1,21 @@
+package forms
+
+import play.api.libs.json.{JsObject, JsValue}
+
+
+class DateWithDaysRule(dateName: String, dateRules: Seq[FieldRule], daysName: String, daysRules: Seq[FieldRule]) extends FieldRule {
+  override def validateOnPreview: Boolean = true
+
+  override def validate(value: JsValue): Seq[String] = {
+    val o = value.validate[JsObject].asOpt.getOrElse(JsObject(Seq()))
+
+    val dateValue =(o \ dateName).toOption.getOrElse(JsObject(Seq()))
+    val daysValue = (o \ daysName).toOption.getOrElse(JsObject(Seq()))
+
+    //dateRules.flatMap()
+
+    ???
```

Comment:
Just pushed a change to remove the FieldRules. They've been replaced with FieldValidators